client/galaxy/scripts/mvc/history/hdca-li-edit.js

```diff
@@ -25,22 +32,46 @@ var HDCAListItemEdit = _super.extend(
             return _super.prototype._renderPrimaryActions.call(this).concat([this._renderDeleteButton()]);
         },
 
-        /** Render icon-button to delete this collection. */
         _renderDeleteButton: function() {
-            var deleted = this.model.get("deleted");
-            return faIconButton({
-                title: deleted ? _l("Dataset collection is already deleted") : _l("Delete"),
-                classes: "delete-btn",
-                faIcon: "fa-times",
-                disabled: deleted,
-                onclick: () => {
-                    // ...bler... tooltips being left behind in DOM (hover out never called on deletion)
-                    this.$el.find(".icon-btn.delete-btn").trigger("mouseout");
-                    this.model["delete"]();
-                }
-            });
+            return $(`
+                <div class="delete-dropdown dropdown">
+                    <a class="delete-btn icon-btn" title="${_l('Delete')}" data-toggle="dropdown">
+                        <span class="fa fa-times"></span>
+                    </a>
+                    <ul class="dropdown-menu pull-right" role="menu" aria-labelledby="dLabel">
```

Comment:
the aria here might be broken copypasta, what is dLabel?

Reply:
copypasta from metadata file downloads

Reply:
It is [supposed to be](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) a list of element ids. I think it is broken there too.